adoc/bp_chap_getting_started_with_salt_formulas.adoc

```diff
@@ -16,22 +16,25 @@ Formulas are collections of Salt States that have been pre-written by other Salt
 Formulas allow for reliable reproduction of a specific configuration again and again.
 Formulas can be installed from RPM packages or an external git repository.
 
-The following list will help you in making the decision to use a state or a formula.
+The following list will help you whether using a state or a formula.
```

Comment:
Sorry, didn't notice this the first time around. You've removed the verb from this sentence. Try:
`This list will help you decide whether to use a state or a formula`